website/docs/d/supervisor_zone.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_supervisor_zone"
+sidebar_current: "docs-vcfa-data-source-supervisor-zone"
+description: |-
+  Provides a data source to read Supervisor Zones in VMware Cloud Foundation Automation.
+---
+
+# vcfa\_supervisor\_zone
+
+Provides a data source to read Supervisor Zones in VMware Cloud Foundation Automation.
+
+## Example Usage
+
+```hcl
+data "vcfa_vcenter" "one" {
+  name = "vcenter-one"
+}
+
+data "vcfa_supervisor" "one" {
+  name       = "my-supervisor-name"
+  vcenter_id = data.vcfa_vcenter.one.id
+}
+
+data "vcfa_supervisor_zone" "one" {
+  supervisor_id = data.vcfa_supervisor.one.id
+  name          = "domain-c8"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `supervisor_id` - (Required) ID of parent Supervisor
+* `name` - (Required) The name of Supervisor Zone
+
+## Attribute Reference
+
+* `vcenter_id` - vCenter server ID that contains this Supervisor
+* `region_id` - Region ID that consumes this Supervisor
+* `cpu_capacity_mhz` - The CPU capacity (in MHz) in this zone. Total CPU consumption in this zone
+  cannot cross this limit.
+* `cpu_used_mhz` - Total CPU used (in MHz) in this zone.
```

Comment:
I have this HCL:

```
data "vcfa_supervisor_zone" "t" {
  supervisor_id = data.vcfa_supervisor.t.id
  name          = var.supervisor_zone_name
}

output "cpu_capacity_mhz" {
  value = data.vcfa_supervisor_zone.t.cpu_capacity_mhz
}
output "cpu_used_mhz" {
  value = data.vcfa_supervisor_zone.t.cpu_used_mhz
}
output "memory_capacity_mib" {
  value = data.vcfa_supervisor_zone.t.memory_capacity_mib
}
output "memory_used_mib" {
  value = data.vcfa_supervisor_zone.t.memory_used_mib
}
```

Suspiciously, `cpu_used_mhz` is showing up as zero:

![Screenshot 2025-01-22 at 11 47 55](https://github.com/user-attachments/assets/ea9c9f1f-72aa-4e30-bfc9-31a73c6aceb7)

Reply:
It seems API returns 0 as well, worth checking separately.